### ISSUE TITLE
Allow running test "under cursor"

### ISSUE THREAD

**Issue by Gert-dev** [yacut/tester#27]
Sometimes a test is failing or you're writing a new test and it can be handy to run just that test in isolation, i.e. because running the entire test suite takes a long time, or just because you may be stepping through your code and don't want any of the breakpoints triggering because of other tests.

With this feature you would be able to simply run the test that the cursor is in. This could be via the gutter, a keyboard shortcut, [an intention](https://github.com/steelbrain/intentions) or some other means.

Thanks!

**Comment by bronson** [comment]
Agreed, this would be nicer than splattering `test.only`, `fdescribe`, and `fffit` through my test files.  It would require buy-in from the test runners of course.
